SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -0,0 +1,545 @@
+			bSuccess = true;
+		}
+		else
+		{
+			UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Local deployment creation failed. Deployment status: %s"), *DeploymentStatus);
+		}
+	}
+	else
+	{
+		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("'status' does not exist in Json result from 'spot create': %s"), *SpotCreateResult);
+	}
+
+	return bSuccess;
+}
+
+bool FLocalDeploymentManager::TryStopLocalDeployment()
+{
+	if (!bLocalDeploymentRunning || LocalRunningDeploymentID.IsEmpty())
+	{
+		UE_LOG(LogSpatialDeploymentManager, Verbose, TEXT("Tried to stop local deployment but no active deployment exists."));
+		return false;
+	}
+
+	bStoppingDeployment = true;
+
+	FString SpotExe = GetSpotExe();
+	FString SpotDeleteArgs = FString::Printf(TEXT("alpha deployment delete --id=%s --json"), *LocalRunningDeploymentID);
+	const FString SpatialDirectory = FSpatialGDKServicesModule::GetSpatialOSDirectory();
+
+
+	FString SpotDeleteResult;
+	FString StdErr;
+	int32 ExitCode;
+	FPlatformProcess::ExecProcess(*SpotExe, *SpotDeleteArgs, &ExitCode, &SpotDeleteResult, &StdErr, *SpatialDirectory);
+	bStoppingDeployment = false;
+
+	if (ExitCode != ExitCodeSuccess)
+	{
+		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Failed to stop local deployment! %s"), *StdErr);
+	}
+
+	bool bSuccess = false;
+
+	TSharedPtr<FJsonObject> SpotJsonResult;
+	bool bPasingSuccess = ParseJson(SpotDeleteResult, SpotJsonResult);
+	if (!bPasingSuccess)
+	{
+		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Json parsing of spot delete result failed."));
+	}
+
+	const TSharedPtr<FJsonObject>* SpotJsonContent = nullptr;
+	if (bPasingSuccess && !SpotJsonResult->TryGetObjectField(TEXT("content"), SpotJsonContent))
+	{
+		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("'content' does not exist in Json result from 'spot delete': %s"), *SpotDeleteResult);
+		bPasingSuccess = false;
+	}
+
+	FString DeploymentStatus;
+	if (bPasingSuccess && SpotJsonContent->Get()->TryGetStringField(TEXT("status"), DeploymentStatus))
+	{
+		if (DeploymentStatus == TEXT("STOPPED"))
+		{
+			UE_LOG(LogSpatialDeploymentManager, Log, TEXT("Successfully stopped local deplyoment"));
+			LocalRunningDeploymentID.Empty();
+			bLocalDeploymentRunning = false;
+			bSuccess = true;
+		}
+		else
+		{
+			UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Stopping local deployment failed. Deployment status: %s"), *DeploymentStatus);
+		}
+	}
+	else
+	{
+		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("'status' does not exist in Json result from 'spot delete': %s"), *SpotDeleteResult);
+	}
+
+	return bSuccess;
+}
+
+bool FLocalDeploymentManager::TryStartSpatialService()
+{
+	if (bSpatialServiceRunning)
+	{
+		UE_LOG(LogSpatialDeploymentManager, Verbose, TEXT("Tried to start spatial service but it is already running."));
+		return false;
+	}
+
+	bStartingSpatialService = true;
+
+	FString SpatialExe = TEXT("spatial.exe");
```

Comment:
Could you make this FString a static const or just a const within FLocalDeploymentManager?